sdk/state/state.go

```diff
@@ -150,36 +174,6 @@ func (c *Channel) verifySigned(tx *txnbuild.Transaction, sigs []xdr.DecoratedSig
 	return false, nil
 }
 
-func (c *Channel) CloseStart(iterationNumber int) error {
```

Comment:
this is removing functions that were originally added as part of the skeleton sdk. Now can be removed as the actual functions are being implemented